images/image-awaiter/Dockerfile

```
@@ -1,15 +1,17 @@
 # compile the code to an executable using an intermediary image
-FROM golang:1.9.2
-
-COPY *.go /go/
-RUN CGO_ENABLED=0 GOOS=linux go build -ldflags '-w -s' -installsuffix cgo -a -o /go/image-awaiter /go/*.go
+FROM golang:1
```

```suggestion
FROM golang:1.15
```

I'm not too familiar with go, but you specify `go 1.15` within go.mod, so I assume it can make sense to connect that with this builder version.

---

Since the go runtime follows semver, the contract is that I should be able to compile code targeting `1.15` against Go runtime version `1.34`

That may or may not be true for 3rd party packages, so I think it's fine to say 

"Use the latest official Go runtime to compile/build, but when pulling in 3rd party deps, be explicit in case one of those declares an incompatibility"